device/pcie/pci_device.cpp

```diff
@@ -197,7 +196,7 @@ void PCIDevice::setup_device() {
     this->pcie_device_id = get_pcie_info(this->device_id, "pcie_device_id");
     this->pcie_revision_id = get_pcie_info(this->device_id, "revision");
     this->arch = detect_arch(pcie_device_id, pcie_revision_id);
-    this->architecture_implementation = tt::umd::architecture_implementation::create(static_cast<tt::umd::architecture>(arch));
+    this->architecture_implementation = tt::umd::TTDevice::create(static_cast<tt::umd::architecture>(arch));
```

Comment:
`this->tt_device`?

Reply:
This will disappear with the next actual refactor, this was just a rename of the type so the following refactor is clearer. PCIDevice shouldn't hold TTDevice, but vice-versa, according to the redesign diagram.